cmd/mc-router/main.go

```diff
@@ -184,6 +201,15 @@ func main() {
 			server.NewWebhookNotifier(config.Webhook.Url, config.Webhook.RequireUser))
 	}
 
+	var cacheInterval time.Duration
+	if config.AutoScale.CacheStatus {
+		cacheInterval, err = time.ParseDuration(config.AutoScale.CacheStatusInterval)
```

Comment:
It seems like this config processing and then ultimately `cacheInterval` should be fed into server.ConnectorConfig